bills-app/src/main/webapp/script.js

```diff
@@ -12,17 +12,788 @@
+                "introduced_date": "2017-11-03",
+                "active": false,
+                "last_vote": null,
+                "house_passage": null,
+                "senate_passage": null,
+                "enacted": null,
+                "vetoed": null,
+                "cosponsors": 8,
+                "cosponsors_by_party": {
+                    "D": 8
+                },
+                "committees": "House Natural Resources Committee",
+                "committee_codes": ["HSII"],
+                "subcommittee_codes": [],
+                "primary_subject": "Energy",
+                "summary": "",
+                "summary_short": "",
+                "latest_major_action_date": "2017-11-03",
+                "latest_major_action": "Referred to the House Committee on Natural Resources."
+            },
+            {
+                "bill_id": "hr4242-115",
+                "bill_slug": "hr4242",
+                "bill_type": "hr",
+                "number": "H.R.4242",
+                "bill_uri": "https://api.propublica.org/congress/v1/115/bills/hr4242.json",
+                "title": "To amend title 38, United States Code, to establish a permanent VA Care in the Community Program, and for other purposes.",
+                "short_title": "To amend title 38, United States Code, to establish a permanent VA Care in the Community Program, and for other purposes.",
+                "sponsor_title": "Rep.",
+                "sponsor_id": "R000582",
+                "sponsor_name": "Phil Roe",
+                "sponsor_state": "TN",
+                "sponsor_party": "R",
+                "sponsor_uri": "https://api.propublica.org/congress/v1/members/R000582.json",
+                "gpo_pdf_uri": null,
+                "congressdotgov_url": "https://www.congress.gov/bill/115th-congress/house-bill/4242",
+                "govtrack_url": "https://www.govtrack.us/congress/bills/115/hr4242",
+                "introduced_date": "2017-11-03",
+                "active": false,
+                "last_vote": null,
+                "house_passage": null,
+                "senate_passage": null,
+                "enacted": null,
+                "vetoed": null,
+                "cosponsors": 23,
+                "cosponsors_by_party": {
+                    "R": 13,
+                    "D": 10
+                },
+                "committees": "House Veterans&#39; Affairs Committee",
+                "committee_codes": ["HSVR"],
+                "subcommittee_codes": [],
+                "primary_subject": "",
+                "summary": "",
+                "summary_short": "",
+                "latest_major_action_date": "2017-11-03",
+                "latest_major_action": "Referred to the House Committee on Veterans' Affairs."
+            }
+        ]
+    }]
 }
+
+// Testing the visualization of the response data on the front end
+const getBills = () => {
```

Comment:
Can you change the function definition/encapsulate this in a function renderBills(bills) where bills is an array of the JSON bill objects? This way, we can pass in any set of bills, from upcoming bills to bills that match a query term, etc. This would support usage like this:

on page load, call getUpcomingBills() and pass that response to renderBills()
if someone enters something in our search bar, call searchBills() and pass that response to renderBills()

This way, renderBills is solely focused on just displaying some given bills. Does this make sense?